lib/form.js

```diff
@@ -1,6 +1,8 @@
 'use strict'
 
 const Emitter = require('component-emitter')
+const modalHtml = require('./modal.html')
```

Comment:
The html and less are in separate file for bette  readability. Everything is compiled together at compile time.